farnaz/challenge2.html

```diff
@@ -0,0 +1,24 @@
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+</head>
+
+<body>
+    <script>
+        //convert color numbers from RGB into HEX code
+        function rgbToHex(red, green, blue) {
+            //convert each color component to a 2-digit hex string and concatenate them
+            const hexString = ((red << 16) + (green << 8) + blue).toString(16).padStart(6, '0');
```

Comment:
Could you please explain what does the `padStart` does?

Reply:
It pads the string with another string (multiple times) until it reaches a given length.
The first value is for the length and the second value is for the string that pads the main string.